### PR TITLE
Make working with UIView frame super easy

### PR DESCRIPTION
**Pull request by aashishdhawan**
Adds more sugar to UIView

### CODE REVIEW

File: Source/iOS/Extensions/UIView+Frame.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public extension UIView {
+    
+    public var height: CGFloat {
```
Comment (onmyway133) [inline Source/iOS/Extensions/UIView+Frame.swift]: Sounds like `ViewUtils` :relaxed:

Comment (aashishdhawan) [inline Source/iOS/Extensions/UIView+Frame.swift]: May be. I was just going through the code of Example `Apple News` in `Spots`. Just saw this 

```swift
    featuredImage.frame.size.width = contentView.frame.width - 30
    featuredImage.frame.size.height = 200
    featuredImage.frame.origin.y = 15
    featuredImage.frame.origin.x = 15

    titleLabel.text = item.title
    titleLabel.frame.size.width = contentView.frame.width - 30
    titleLabel.sizeToFit()
    titleLabel.frame.origin.x = 15
    titleLabel.frame.origin.y = featuredImage.frame.maxY + 15

    introLabel.text = item.subtitle
    introLabel.frame.size.width = contentView.frame.width - 30
    introLabel.sizeToFit()
    introLabel.frame.origin.x = 15
    introLabel.frame.origin.y = titleLabel.frame.maxY + 15
```

What do you think @onmyway133 ? Is this not fit for `Sugar` ?

Comment (onmyway133) [inline Source/iOS/Extensions/UIView+Frame.swift]: @aashishdhawan I like it, been using ViewUtils since ObjC :smile:

Comment (aashishdhawan) [inline Source/iOS/Extensions/UIView+Frame.swift]: yes @onmyway133  i also got this idea from some else. Just standing on the [shoulders of giants](https://en.wikipedia.org/wiki/Standing_on_the_shoulders_of_giants)

**Comment by zenangst** [comment]
This is nifty, I really like this!

**Comment by zenangst** [comment]
This got me thinking, what about adding this to `CALayer` as well?

**Comment by aashishdhawan** [comment]
We can..but i think we need to put a boundary somewhere. I actually do not like to add too many things in `Sugar`. Just most awesome things which are needed frequently. if you think something like this is needed on `CALayer` lets go ahead.
Actually i was think to add `makeRoundedCorner` and `addShadow` like methods.

**Comment by zenangst** [comment]
This is true, so far I've only found one place where we use `CALayer`. If the two had a shared protocol we could have added it there instead but they don't as far as I can see.

**Pull request merged**